frontend/packages/eslint-plugin-console/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "eslint-plugin-console",
+  "version": "0.0.0-fixed",
+  "description": "Console ESLint rules and configs.",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint . --max-warnings 0 --format codeframe",
+    "prettier": "eslint --format codeframe . --fix",
+    "prettier-check": "eslint --print-config ./lib/config/prettier.js | eslint-config-prettier-check"
+  },
+  "dependencies": {
+    "@typescript-eslint/eslint-plugin": "1.9.0",
+    "@typescript-eslint/parser": "1.9.0",
```

Comment:
Do we need this dependency, given that we'll use `typescript-eslint-parser` until prettier/prettier-eslint#201 gets fixed?

Reply:
Right now yes because the root eslint files is using it. And I moved all eslint related deps to this package.

Reply:
> root eslint files is using it

Ah yes, I forgot :) OK, agreed on putting all ESLint deps here.